flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -155,7 +156,11 @@ class TestName extends BeforeEachCallback {
       }
       methodName = s"${context.getTestMethod.get().getName}$displayName"
     } else {
-      methodName = context.getTestMethod.get().getName
+      if (AnnotationSupport.isAnnotated(context.getTestMethod, classOf[ParameterizedTest])) {
```

Comment:
```suggestion
      if (AnnotationSupport.isAnnotated(context.getTestMethod, classOf[ParameterizedTest])
      || (AnnotationSupport.isAnnotated(context.getTestMethod, classOf[TestTemplate])) {
```
May be also include `TestTemplate` ?